JS.md

```diff
@@ -62,51 +62,87 @@ The following intrinsic objects are added:
 
 #### `WebAssembly.validate`
 
-The `validate` function has the signature:
+The `validate` function is overloaded based on types of its arguments.
+If none of the following overloads match, then the returned `Promise` is
+[rejected](http://tc39.github.io/ecma262/#sec-rejectpromise)
+with a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror).
 
 ```
 Boolean validate(BufferSource bytes)
+
+Boolean validate(Response source)
```

Comment:
`validate` is only useful for feature testing, so I'm not sure it's worth adding a whole Response overload and Promise type for this.  I don't think anyone should be calling validate on a whole codebase.  Furthermore, validation, unlike compilation, is crazy fast.